ray-operator/controllers/ray/common/pod.go

```diff
@@ -125,6 +125,17 @@ func configureGCSFaultTolerance(podTemplate *corev1.PodTemplateSpec, instance ra
 					Name:  utils.RAY_REDIS_ADDRESS,
 					Value: options.RedisAddress,
 				})
+				if options.RedisUsername != nil {
+					// If `GcsFaultToleranceOptions.RedisUsername` is set, it will be put into the
+					// `REDIS_USERNAME` environment variable later. Here, we use `$REDIS_USERNAME` in
+					// rayStartParams to refer to the environment variable.
```

Comment:
Add a comment: "Note that `redis-username` will be supported starting from Ray 2.41."